src/hook/useStepper.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { StepId, StepStatus } from 'ui/index'
+import { StepperState, StepperAction, stepperReducer } from './useStepper'
+import { List as ImmutableList, OrderedMap } from 'immutable'
+
+describe('Considering the stepperReducer function', () => {
```

Comment:
I wouldn't have isolated the `stepperReducer` function as SUT, because, here, you want to test the whole hook: `useStepper` not the `reducer` only.

For that, you need to instantiate before each test a new instance of `useStepper` hook by providing data args. Then, in the test itself, you will dispatch some actions and make an assertion on the expected result.

By doing that, you will test the whole behavior, from the `useStepper` initialization to the state mutation.